docs/UserGuide.md

```diff
@@ -593,20 +633,22 @@ Here are some potential error messages that you may receive and here's how to so
 
 #### View all leaves: `find-all-leave`
 
-Use the `find-all-leave` command to view all leave records currently stored in HRMate.
+Use the `find-all-leave` command to view all leave applications currently stored in HRMate.
 
 1. Type in the following command in the command box:
    `find-all-leave`
-2. Press `Enter` to execute the command. You should see all the leave records in HRMate.
+2. Press `Enter` to execute the command. You should see all the leave applications in HRMate.
+
+<div style="page-break-after: always;"></div>
 
 
-### Edit a leave record
+### Edit a leave application
 
-HRMate offers different commands for editing leave records. [`approve-leave`](#approve-leave-record-by-index-approve-leave) and [`reject-leave`](#reject-leave-record-by-index-reject-leave) would approve and reject leave records respectively while [`edit-leave`](#editing-a-leave-record-edit-leave) would edit the leave record comprehensively.
+HRMate offers different commands for editing leave applications. [`approve-leave`](#approve-leave-record-by-index-approve-leave) and [`reject-leave`](#reject-leave-record-by-index-reject-leave) would approve and reject leave applications respectively while [`edit-leave`](#editing-a-leave-record-edit-leave) would edit the leave application comprehensively.
```

Comment:
Similarly i think the links here might be broken due to the renaming of leave records into leave applications